### ISSUE TITLE
Launch Jupyter Console gives error in directory with space

### ISSUE THREAD

**Issue by dnowacki-usgs** [lgeiger/hydrogen-launcher#8]
Similar to #4, I'm again getting an error when working in a directory with a space, this time for `Hydrogen Launcher: Launch Jupyter Terminal`. I think it used to work without the error (#4 suggests as much). hydrogen-launcher 1.10.0, Atom 1.12.7, OS X 10.10.5.

After running `Hydrogen Launcher: Launch Jupyter Terminal` I get the following:
```
/Users/dnowacki/.atom/packages/hydrogen-launcher/node_modules/term-launcher/cmd-script.sh ; exit;
/Users/dnowacki/.atom/packages/hydrogen-launcher/node_modules/term-launcher/cmd-script.sh: line 2: cd: /Users/dnowacki/Documents/Space: No such file or directory
Jupyter Console 4.0.3

[ZMQTerminalIPythonApp] Loading IPython extension: storemagic

In [1]: pwd
Out[2]: u'/Users/dnowacki/Documents/Space Dir/py'
```
i.e. I get a no such file or directory error when launching the terminal, but the Jupyter console starts in the correct directory (with space) anyway.

**Comment by lgeiger** [comment]
I just published `v1.1.1` it should be fixed there 🚀

**Comment by dnowacki-usgs** [comment]
Nice, thanks 👍
